### ISSUE TITLE
Hi! How does this work? The list shows up blank for me.

### ISSUE THREAD

**Issue by ghost** [gd-irc/Transmisia#1]

**Comment by SoniEx2** [comment]
Hm? https://github.com/gorhill/uBlock/wiki/Filter-lists-from-around-the-web

**Comment by ghost** [comment]
Thank You. So just confirming I import the text file and that's all?

**Comment by SoniEx2** [comment]
yeah

**Comment by ghost** [comment]
Thanks! How do I see a list of the transphobic/transmisia sites which are being blocked?

**Comment by SoniEx2** [comment]
the eye icon beside it should show the full list, which currently has 1 entry

**Comment by ghost** [comment]
Hi sorry I think I'm confused as I thought it would be a list of sites

**Comment by SoniEx2** [comment]
it's an ublock origin filter list.

**Comment by konomikitten** [comment]
The read me having details on how to import the list and what extension is needed might be better for less experienced users. Also detailing how it works and what to expect when they encounter a block etc.

**Comment by ghost** [comment]
Thanks @konomikitten -- would you be able to explain/show how to use this?

**Comment by konomikitten** [comment]
> Thanks @konomikitten -- would you be able to explain/show how to use this?

Considering the list only has one entry unless I'm mistaken, doesn't seem like there is much point using it at the moment.

**Comment by SoniEx2** [comment]
yeah, we didn't have a good way to seed it but we figured we'd start somewhere. pull requests are welcome tho.
